CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ If the contribution doesn't meet these criteria, a maintainer will discuss it wi
 * Make your changes
 * When committing your changes, make sure to start the commit message with `#<issue-number>` (e.g. `git commit -m '#12 - New Feature added'`)
 * Push your new branch to your own fork into the same remote branch (e.g. `git push origin 12-new-feature`)
-* On Gitlab goto the [pull request page](https://guides.github.com/activities/forking/#making-a-pull-request) on your own fork and create a merge request to this reposistory
+* On Github go to the [pull request page](https://guides.github.com/activities/forking/#making-a-pull-request) on your own fork and create a merge request to this repository
```

Comment:
You don't have to go to your own fork to do this, it will pop-up on the main repo when you go into it.
merge request -> pull request?

Reply:
True, but I found that harder to explain then this, and this works as well :-)